src/components/Wallets.jsx

```diff
@@ -54,16 +56,20 @@ export default function Wallets({ currentWallet, startWallet, stopWallet }) {
       }
 
       setAlert(null)
+      setIsUnlocking(true)
       try {
         const res = await Api.postWalletUnlock({ walletName }, { password })
         const body = await (res.ok ? res.json() : Api.Helper.throwError(res))
 
+        setIsUnlocking(false)
+
         const { walletname: unlockedWalletName, token } = body
         startWallet(unlockedWalletName, token)
         navigate(routes.wallet)
       } catch (e) {
         const message = e.message.replace('Wallet', walletName)
         setAlert({ variant: 'danger', dismissible: false, message })
+        setIsUnlocking(false)
```

Comment:
Maybe put it in a `finally` clause?

Reply:
I think this is not possible due to `navigate` unmounting the component. Will try and report back.

Reply:
Funny how I get 
```
Warning: Can't perform a React state update on an unmounted component. This is a no-op, but it indicates a memory leak in your application. To fix, cancel all subscriptions and asynchronous tasks in a `useEffect` cleanup function.
```
in Chrome, but not in Firefox. It seems I have not fully grasp the component lifecycle as of yet. Would have thought it is the same in both browsers. Is it okay to leave it unchanged?

Reply:
> I think this is not possible due to navigate unmounting the component. 

Makes sense! ✔️